packages/jest-types/__typetests__/jest.test.ts

```diff
@@ -667,6 +667,11 @@ expect(jest.retryTimes(3, {logErrorsBeforeRetry: 'all'})).type.toRaiseError();
 expect(jest.retryTimes({logErrorsBeforeRetry: true})).type.toRaiseError();
 expect(jest.retryTimes(3, {waitBeforeRetry: 1000})).type.toEqual<typeof jest>();
 expect(jest.retryTimes(3, {waitBeforeRetry: true})).type.toRaiseError();
+expect(jest.retryTimes(3, {retryImmediately: true})).type.toEqual<
```

Comment:
I think `retryImmediately` reads better and looks good in this context. Let’s see what Simen will say.

Thanks for type test!